resources/views/masjid_profile/edit.blade.php

```diff
@@ -12,6 +12,16 @@
                     {!! FormField::text('masjid_name', ['required' => true, 'value' => old('masjid_name', Setting::get('masjid_name')), 'label' => __('masjid_profile.name')]) !!}
                     {!! FormField::textarea('masjid_address', ['required' => true, 'value' => old('masjid_address', Setting::get('masjid_address')), 'label' => __('masjid_profile.address')]) !!}
                     {!! FormField::text('masjid_google_maps_link', ['value' => old('masjid_google_maps_link', Setting::get('masjid_google_maps_link')), 'label' => __('masjid_profile.google_maps_link')]) !!}
+                    <div class="form-group" id="masjid-logo">
+                        <label class="form-label" for="masjid_logo_image">{{__('masjid_profile.masjid_logo')}}</label>
+                        <div class="custom-file">
+                          <input type="file" class="custom-file-input image" id="masjid_logo_image" name="masjid_logo_image">
+                          <label class="custom-file-label" for="masjid_logo_image">{{__('masjid_profile.choose_file')}}</label>
+                        </div>
+                        @if (Setting::get('masjid_logo_path'))
+                            <img id="masjid_logo_image_show" class="img-fluid mt-2" src="{{ Storage::url(Setting::get('masjid_logo_path'))}}" alt="{{ Setting::get('masjid_name') ?? 'buku masjid'}}">
```

Comment:
Di fitur setting itu kita bisa kasih default value mas: `Setting::get('key', 'default_value');`. Jadi default valuenya ambil nama masjid dari config saja.

```suggestion
                            <img id="masjid_logo_image_show" class="img-fluid mt-2" src="{{ Storage::url(Setting::get('masjid_logo_path'))}}" alt="{{ Setting::get('masjid_name', config('masjid.name')) }}">
```